integration_tests/helpers/config.js

```diff
@@ -60,21 +59,23 @@ function baseEnvs(peerSeeds = [])
      return envs;
 }
 
-function createEnv(nodeFile="newnodeid.json",walletGrpcAddress, walletGrpcPort, baseNodeGrpcAddress="127.0.0.1",baseNodeGrpcPort="8080", baseNodePort="8080",proxyFullAddress="127.0.0.1:8080",options, peerSeeds=[]) {
+function createEnv(name="config_identity",isWallet=false, nodeFile="newnodeid.json",walletGrpcAddress="127.0.0.1", walletGrpcPort="8082", walletPort="8083", baseNodeGrpcAddress="127.0.0.1", baseNodeGrpcPort="8080", baseNodePort="8081",proxyFullAddress="127.0.0.1:8084",options, peerSeeds=[]) {
```

Comment:
Code can be simplified.

`isWallet=false,` is not needed here, whereby `walletPort="8083",` and `baseNodePort="8081",` can be combined into one variable called `nodePort="8081",` or similar.